diffgraph/structural.py

```
@@ -0,0 +1,408 @@
+
+
+def _worktree_bytes(repository: str, entry: SnapshotEntry) -> Optional[bytes]:
+    if entry.new_path is None:
+        return None
+    path = os.path.join(repository, entry.new_path)
+    before = os.stat(path, follow_symlinks=False)
+    with open(path, "rb") as handle:
+        content = handle.read()
+        opened = os.fstat(handle.fileno())
+    after = os.stat(path, follow_symlinks=False)
+    identity = lambda value: (value.st_dev, value.st_ino, value.st_size, value.st_mtime_ns)
```

Ruff (0.16.0)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `identity` as a `def`

(E731)